app/backend/db/crud/date_datas.py

```diff
@@ -29,11 +32,78 @@ def get_multiple_users_dates_data(
     users_id: List[int],
     start_date: date,
     end_date: date = None
-) -> List[schemas.DateResponse]:
+) -> List[schemas.RangeDatesResponse]:
+
     return [
         {
             'user_id': user_id,
             'data': get_dates_data(db, user_id, start_date, end_date)
         }
```

Comment:
isnt it better to return a dict of key = user_id, value = get_dates_data?

Reply:
IDK I don't mind

Reply:
done